udapi/block/write/conllu.py

```diff
@@ -84,7 +87,8 @@ def process_document(self, document):
                         self.file_handler.write('\t'.join([value for value in values]))
                         self.file_handler.write('\n')
 
-                    self.file_handler.write("\n")
+                    if number_of_written_bundles != len(document.bundles) - 1:
```

Comment:
according to CoNLL-U specification, each tree (including the last one) must end with an empty line